hw/src/lib.rs

```diff
@@ -181,16 +183,18 @@ impl From<trezor::Error> for Error {
 }
 
 impl From<libusb::Error> for Error {
-	fn from(err: libusb::Error) -> Error {
+	fn from(err: libusb::Error) -> Self {
 		Error::Usb(err)
 	}
 }
 
 /// Specifies the direction of the `HardwareWallet` i.e, whether it arrived or left
 #[derive(Debug, Copy, Clone)]
 pub enum DeviceDirection {
-    Arrived,
-    Left,
+	/// Device arrived
```

Comment:
As in "user pulled out the wallet from the usb port"? Maybe be explicit here and say `Device became available`/`Device was removed from the system`?

Reply:
@dvdplm do you want `s/Arrived/Available/ s/Left/Unavailable` renamed completely including the enum patterns? 

Happy to change, I have no strong opinions here!

Reply:
I think that would make sense, yes, but my grumble was about the doc-comment. :)

Reply:
Ok, I having second thoughts on this because `DeviceDirection` doesn't actually indicate whether a device became available or unavailable. Essentially, it only tells we got a `notification` that a `USB device` was connected or disconnected with manufacturer ID (Ledger or Trezor). To actually say that a device become available/unavailable we need to do additional checks, for instance, request an Ethereum address and so on!

So, I will keep this! Ok?!